IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -159,6 +159,7 @@ @interface MSIDMacKeychainTokenCache ()
 
 @property (readwrite, nonnull) NSString *keychainGroup;
 @property (readwrite, nonnull) NSDictionary *defaultCacheQuery;
+@property (readwrite, nonnull) NSString *lastModificationApp;
```

Comment:
since this is a static thing, let's call it appIdentifier?

Reply:
updated